DynamicMeter.c

```diff
@@ -48,8 +48,8 @@ void DynamicMeters_delete(Hashtable* dynamics) {
 }
 
 typedef struct {
-   unsigned int key;
    const char* name;
+   unsigned int key;
```

Comment:
As this key refers to a hashmap key internally, it's type should be `ht_key_t`, similar to the `DynamicIterator` type defined in `DynamicScreen.c`.

@natoscott I think that's something for you to take a look at.

Also given the semantics, having the key field first for this struct is more intuitive (and AFAICS, there'sonly few of these structs worked on anyway.

Reply:
@BenBE I've opened #1703 to clean this up.